server/DatabaseHandler.py

```diff
@@ -202,7 +234,7 @@ def get_new_cse_transcripts_for_user(self, user_id, delete_after=False):
                     backslide_words = ' '.join(backslide_word_list[-(self.backslide-len(backslide_word_list)):])
 
                     words_from_start = first_transcript['text'][start_index:].strip()
-                    first_transcript['text'] = backslide_words + " " + words_from_start
+                    first_transcript['text'] = backslide_words + " " + words_from_start if words_from_start else backslide_words
```

Comment:
didn't know python would parse "" as false and "123" as true, not really glaring but could be more explicit to improve readability